opam

```diff
@@ -3,17 +3,16 @@ maintainer: "xen-api@lists.xen.org"
 authors: "xen-api@lists.xen.org"
 homepage: "https://github.com/xapi-project/xenopsd"
 build: [
-	["oasis" "setup"]
 	["./configure"]
-  ["ocaml" "setup.ml" "-configure" "--disable-libvirt"]
 	[make]
 ]
 install: [
- 	["ocaml" "setup.ml" "-install"]
+        ["make" "setup.bin"]
```

Comment:
The next step fails, because `setup.data` is also required. We could change this to:
```
["make" "setup.bin" "setup.data"]
```
In general, it's easier to send these PRs to `xs-opam` first, because that will test the installation and uninstallation of each package, but this repo doesn't have opam-based Travis builds.